### PR TITLE
minesweeper: split into introduction and instructions

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/minesweeper/introduction.md
Comment (ErikSchierboom) [inline exercises/minesweeper/introduction.md:1]: Do we want a story here?

Comment (BethanyG) [reply exercises/minesweeper/introduction.md:1]: I don't think it needs one, really.  But maybe we add a link ([Minesweeper](https://en.wikipedia.org/wiki/Minesweeper_(video_game))) for those who might not know about the game?

**Review by BethanyG — APPROVED** [review]

**Comment by BethanyG** [comment]
Tried re-running the markdown lint, but it DOES NOT like that space with the code highlight.  Is there a way to put a skip in?

File: exercises/minesweeper/instructions.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Instructions
+
+Your task is to add the mine counts to empty squares in a completed Minesweeper board.
+The board itself is a rectangle composed of squares that are either empty (`' '`) or a mine (`'*'`).
+
+For each empty square, count the number of  mines adjacent to it (horizontally, vertically, diagonally).
```
Comment (ErikSchierboom) [inline exercises/minesweeper/instructions.md:6]:
```suggestion
For each empty square, count the number of mines adjacent to it (horizontally, vertically, diagonally).
```

**Comment by ErikSchierboom** [comment]
> Tried re-running the markdown lint, but it DOES NOT like that space with the code highlight. Is there a way to put a skip in?

I've just removed the space :)

**Review by BNAndras — APPROVED** [review]

**Pull request merged**